api/src/repositories/survey-repository.ts

```diff
@@ -625,6 +686,37 @@ export class SurveyRepository extends BaseRepository {
     return result.id;
   }
 
+  /**
+   * Inserts a new survey_type record.
+   *
+   * @param {number} focal_species_id
+   * @param {number} surveyId
+   * @return {*}  {Promise<void>}
+   * @memberof SurveyRepository
+   */
+  async insertSurveyType(typeId: number, surveyId: number): Promise<void> {
```

Comment:
Thoughts on replacing this with something like:
```typescript
async insertSurveyTypes(typeIds: number[], surveyId: number): Promise<{ survey_type_id: number }[]>
```
Motivation being if we want to attach five types to a survey, we could avoid dispatching five requests to the SQL server.

Reply:
Example:

```typescript
async insertStakeholderPartnerships(
    stakeholderPartners: string[],
    surveyId: number
  ): Promise<{ survey_stakeholder_partnership_id: number }[]> {
    defaultLog.debug({ label: 'insertStakeholderPartnerships', stakeholderPartners, surveyId });
    const queryBuilder = getKnex()
      .table('survey_stakeholder_partnership')
      .insert(
        stakeholderPartners.map((stakeholder: string) => ({
          name: stakeholder,
          survey_id: surveyId
        }))
      )
      .returning('survey_stakeholder_partnership_id');

    const response = await this.connection.knex<{ survey_stakeholder_partnership_id: number }>(queryBuilder);

    if (!response || response.rowCount === 0) {
      throw new ApiExecuteSQLError('Failed to insert survey stakeholder partnerships', [
        'ErrorRepository->insertStakeholderPartnerships',
        'rowCount was null or undefined, expected rowCount != 0'
      ]);
    }

    return response.rows;
  }
```

Reply:
Ya fair point, I was just being fast and lazy. Updated.